packages/beacon-node/src/api/impl/beacon/blocks/index.ts

```diff
@@ -406,7 +406,7 @@ export function getBeaconBlockApi({
       await publishBlock(signedBlockOrContents, opts);
     },
 
-    async getBlobSidecars(blockId) {
+    async getBlobSidecars(blockId, indices = []) {
```

Comment:
I think it would be safer to not set a default value here as the spec states that filtering should be applied if those are provided
> If the `indices` parameter is specified, only the blob sidecars with the specified indices will be returned. There are no guarantees
    for the returned blob sidecars in terms of ordering.